snippets/ponylang-mode/newc

```diff
@@ -3,5 +3,5 @@
 # name: new-create
 # contributor: Sean T. Allen <sean@monkeysnatchbanana.com>
 # --
-new $1create(${2:params}) =>
+new ${1:create}(${2:params}) =>
```

Comment:
I don't think this is doing what you think it is. What do you think the previous was doing, and what is this one doing?

Reply:
it's for customize the constructor name. 
````pony
class Wombat
  let name: String
  var _hunger_level: U64
  var _thirst_level: U64 = 1

  new create(name': String) =>
    name = name'
    _hunger_level = 0

  new hungry(name': String, hunger': U64) =>
    name = name'
    _hunger_level = hunger'
````

Reply:
that means that you have to type "create" which otherwise would have been the default, yes?

what was the previous behavior.

I'm neutral on this change but would want to see it as its own PR, if you think its a good idea, go for it as its own commit and PR.

are you using the snippets much?